dev/make_rc.sh

```diff
@@ -30,10 +30,10 @@ DIR_ROOT="$(cd $(dirname $0);pwd)/.."
 cd $DIR_ROOT
 PHOENIX="$(xmllint --xpath "//*[local-name()='project']/*[local-name()='version']/text()" pom.xml)"
 DIR_REL_BASE=$DIR_ROOT/release
-DIR_REL_ROOT=$DIR_REL_BASE/phoenix-$PHOENIX
-DIR_REL_BIN=phoenix-$PHOENIX-bin
+DIR_REL_ROOT=$DIR_REL_BASE/apache-phoenix-$PHOENIX
```

Comment:
This will be fun for repackagers, but it's something that can be handled with a regex